webapp/src/Service/CheckConfigService.php

```diff
@@ -761,9 +746,11 @@ public function checkSelfRegistration(): array
                     $selfRegistrationCategories[0]->getName());
             } else {
                 $desc .= sprintf("Team categories allowed for self-registered teams: %s.\n",
-                    implode(', ', array_map(function ($category) {
-                        return $category->getName();
-                    }, $selfRegistrationCategories)));
+                    implode(
+                        ', ',
```

Comment:
This would all fit on the same line IMO

Reply:
So
```
implode(', ', array_map(fn($category) => $category->getName(), $selfRegistrationCategories))
```
?

That is 113 characters long which is maybe a bit long, but I'm fine with doing that if others agree (but I think some won't 😛 )

Reply:
We could store the result of `array_map(...)` in a local var and then use that?

Reply:
Yes, but if the rest doesn't like it its fine by me. I'll resolve.